.github/workflows/deploy.yml

```diff
@@ -0,0 +1,56 @@
+name: de-project-vinson
+
+on:
+  push:
+    branches:
+      - '**'
+
+jobs:
+  run-tests:
+    name: run-tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v4
+      - name: Set Up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+      - name: Install pip
+        uses: BSFishy/pip-action@v1
+        with:
+          requirements: requirements.txt
+      - name: Run Create Env
```

Comment:
this needs to be make create-environment and then add a new point  Install requirements and make requirements